projects/plugins/jetpack/extensions/plugins/publicize/index.js

```diff
@@ -60,7 +59,10 @@ const PublicizeSettings = () => {
 
 	return (
 		<PostTypeSupportCheck supportKeys="publicize">
-			<JetpackPluginSidebar>{ children }</JetpackPluginSidebar>
+			<JetpackPluginSidebar>
+				{ children }
+				{ isSocialImageGeneratorAvailable && <SocialImageGeneratorPanel /> }
```

Comment:
I think we need to check if the module is active too, as we did with the panels up a few lines. This way SIG is shown even if the module is active.

![CleanShot 2024-04-15 at 11 27 19 png](https://github.com/Automattic/jetpack/assets/36671565/f53844f9-90dd-46f4-846d-70a526586271)